### PR TITLE
Feature/update metadata to match automated tests

### PR DESCRIPTION
**Pull request by jaredledvina**
This updates the Ansible Galaxy metadata file to only list out the OS'es that we are currently automatically testing via TravisCI and bumps the required Ansible version to 2.3 as we are not actively testing this role for versions older than that. 

I _think_ this makes sense but, am absolutely open to ideas from the community here. 

My current idea is to continue working on the integration tests and as we add more OS'es to that list, continue updating them here.

### CODE REVIEW

**Comment by jaredledvina** [comment]
Also updated the README to match and switched to https://github.com/sensu/sensu-ansible/graphs/contributors for the Contributors.

File: README.md [hunk 1]
```diff
@@ -22,24 +22,25 @@ However, if you want to rely on other roles/management methods to deploy/manage
 
 ## Requirements
 This role requires:
-- Ansible 2.0
+- Ansible 2.3
```
Comment (majormoses) [inline README.md]: I would say this makes it a `### Breaking Changes` in the changelog if this is true. If this is just the tested version they should be called out differently.

Comment (jaredledvina) [reply README.md]: Any thoughts on how to call it out? I'm not actually breaking 2.0 compatibility more-so, we won't actively be testing for it since it's been long out of the Ansible supported status: http://docs.ansible.com/ansible/latest/release_and_maintenance.html#release-status

Comment (majormoses) [reply README.md]: Then the requirements should stay at `2.0` if you are continuing to support it. I would call out tested versions vs supported versions when they differ. I know it seems like bikeshedding but it is an important distinction.

Comment (majormoses) [reply README.md]: You already have a section talking about testing just add something about we test with ansible version `2.3`

Comment (majormoses) [reply README.md]: If it is outside of the support status and there will be no further support on your end for it then yes bump up required. Otherwise people will submit issues and you will just say sorry we dont support EOL versions. I think that there should be a section talking about version support. For example for the plugins: https://github.com/sensu-plugins/documentation/blob/master/docs/FAQ.md#what-is-the-policy-on-supporting-end-of-lifeeol-ruby-versions

Comment (jaredledvina) [reply README.md]: Yeah, cool, so I'd like to start getting this role into the position of 'We actively support the last two stable releases of Ansible' (which would be 2.4 and 2.3 compatibility). However, I have no intention of breaking anything in this role with Ansible 2.2 and older. I feel like limiting the burden on the maintainers to the last two stable releases is fair, and also aligns with Ansible's support on their side. I really wouldn't like to see us attempting to get this role to work on 2.4 while keeping it working on 2.0 for the long term. Additionally, I feel uncomfortable stating that the minimum Ansible version is 2.0 when we only actually verify on each PR that 2.3/2.4 work (although this does admittedly need to be fleshed out more). 

Does that seem fair @majormoses? I can add a blurb about version support like you have for `sensu-plugins` as well to make this explicit for folks.

Comment (majormoses) [reply README.md]: :+1:

**Comment by jaredledvina** [comment]
Updated the README based on the conversation during the Sensu Community Chat yesterday. I also switched out the direct email links to Caelum to just the links to opening an issue/PR and added a link to the community docs on the maintainers. 

I believe this is ready to rock. I plan on including this with the 2.0.0 release as well and calling it out in the CHANGELOG in https://github.com/sensu/sensu-ansible/pull/133

File: meta/main.yml [hunk 2]
```diff
@@ -2,115 +2,19 @@ galaxy_info:
   author: Calum MacRae
```
Comment (majormoses) [inline meta/main.yml:2]: @jaredledvina could this be how to change the owner in ansible galaxy?

File: README.md [hunk 3]
```diff
@@ -60,18 +61,22 @@ Or, passing parameter values:
       - { role: cmacrae.sensu, sensu_master: true, sensu_include_dashboard: true  }
 ```
 
+## Ansible version support
+All changes to this role are actively tested with the last two stable versions of Ansible to ensure compatibility. As such, this role
+only officially supports running with the last two stable releases of Ansible, which aligns with the [Ansible support model](http://docs.ansible.com/ansible/latest/release_and_maintenance.html#release-status).
```
Comment (majormoses) [inline README.md:66]: So I see the min version as 2.3 in the meta, how are you testing multiple versions? Sorry I am still pretty unfamiliar with ansible.

File: meta/main.yml (hunk 2 above)
Comment (jaredledvina) [reply meta/main.yml:2]: Hmmm I might actually be, I'll take a look. (For everyone's context, I'm trying to get this role to show up under the `sensu` author space). Since that will most likely cause people to have to change how they'll pull this role, it'd be a breaking change, so I'll work that out in a separate PR to make the story clean there.

File: README.md (hunk 3 above)
Comment (jaredledvina) [reply README.md:66]: I'm still working on getting official 2.3 testing in Travis. Currently, I am using what Jeff Geerling has in his docker images which is 2.4. I'm hoping to get a solid way to define the Ansible versions along with the OS's. Currently, I am just doing syntax checks locally with 2.3 to make sure (notably there's only 1 large new feature in 2.4 that breaks 2.3 entirely). 

tl;dr - Manually at this point in time, but not for long as I rework the integration tests. That said, this stable is still true, I'm actively ensuring 2.3 support is maintained and will handle any issue with it breaking.
